src/form-builder.tsx

```diff
@@ -1,18 +1,52 @@
-import { useEffect, useState } from "react";
+import { IFrontendEngineData } from "@lifesg/web-frontend-engine/components/types";
```

Comment:
FEE doesnt need sub-directory because it has to load all its components to determine what components to use - there is no savings unlike DS

```suggestion
import { IFrontendEngineData } from "@lifesg/web-frontend-engine";
```